### PR TITLE
Cosmos docs

### PR DESCRIPTION
**Pull request by jayantk**
Add interactive documentation for the cosmos contract to the docs site. 

There are a couple of refactors that are probably worth doing after this change:
1. we should probably extract out contexts for each network so that `GlobalContext` doesn't grow too large.
2. we need to think harder about how the context classes for Example and DynamicCode work, because they're going to grow per-network right now.

I don't want to do those things in this PR though because it's going to make a big diff to the EVM code, and this PR is already pretty big.

### CODE REVIEW

File: components/EvmNetworkSelector.tsx [hunk 1]
```diff
@@ -1,60 +1,19 @@
...
-                    {n}
-                  </button>
-                </Menu.Item>
-              ))}
-            </Menu.Items>
-          </Transition>
-        </>
-      )}
-    </Menu>
+    <Selector
```
Comment (jayantk) [inline components/EvmNetworkSelector.tsx:12]: refactored the common code ^ into the Selector component

File: components/EvmSend.tsx [hunk 2]
```diff
@@ -49,6 +49,7 @@ const EvmSend = ({ functionName, buildArguments, feeKey }: EvmSendProps) => {
   useEffect(() => {
     if (isLoading) {
       setResponsePreface("Loading...");
+      setResponse("");
```
Comment (jayantk) [inline components/EvmSend.tsx:52]: drive-by bug fix: this used to show the old response while loading the new one

File: contexts/GlobalContext.tsx [hunk 3]
```diff
@@ -31,18 +42,20 @@ export interface GlobalContextData {
   pythAddressConfig: PythAddressConfig;
   currentChainConfig?: Chain;
 
-  // Provider for reading from the chain and signer for sending transactions.
-  // If signer is null, the user's wallet is not connected.
+  // Provider for reading from the chain.
   provider: ethers.Provider;
   setProvider: React.Dispatch<React.SetStateAction<ethers.Provider>>;
-  signer: ethers.Signer | undefined;
-  setSigner: React.Dispatch<React.SetStateAction<ethers.Signer | undefined>>;
```
Comment (jayantk) [inline contexts/GlobalContext.tsx:39]: drive-by: this field isn't used anymore since we added wagmi

File: jest.config.mjs [hunk 4]
```diff
@@ -11,7 +11,7 @@ const config = {
   // Add more setup options before each test is run
   setupFilesAfterEnv: ["jest-expect-message"],
   testEnvironment: "jest-environment-jsdom",
-  testTimeout: 10000,
+  testTimeout: 20000,
```
Comment (jayantk) [inline jest.config.mjs:14]: rust tests are slow :(

File: pages/cosmwasm/execute-update-price-feeds.mdx [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+
+<Examples>
+  <Example
+    keyValues={{
+      data: async (ctx) =>
+        (await ctx.getLatestPriceFeed("Crypto.BTC/USD", "default")).vaa,
+      fee: async (ctx) => {
+        // FIXME: this gets the ETH update fee, not the cosmos fee. They're currently the same though.
+        let vaa = (await ctx.getLatestPriceFeed("Crypto.BTC/USD", "evm")).vaa;
+        return await ctx.getEthUpdateFee([vaa]);
```
Comment (jayantk) [inline pages/cosmwasm/execute-update-price-feeds.mdx:40]: this is not great but it's hard to fix right now without adding a bunch of code to Example. I want to kick the can on it for now.

File: pages/documentation/pythnet-price-feeds/cosmwasm.mdx [hunk 6]
```diff
@@ -30,7 +30,7 @@ Pyth is currently available on the following cosmwasm chains:
 | Injective Mainnet | `inj12j43nf2f0qumnt2zrrmpvnsqgzndxefujlvr08`                         |
 | Osmosis Mainnet   | `osmo13ge29x4e2s63a8ytz2px8gurtyznmue4a69n5275692v3qn3ks8q7cwck7`    |
 | Neutron Mainnet   | `neutron1m2emc93m9gpwgsrsf2vylv9xvgqh654630v7dfrhrkmr5slly53spg85wv` |
-| Injective         | `inj18hckkzqf47mdhd734g6papk6wj20y24rm43sk9`                         |
+| Injective Testnet | `inj18hckkzqf47mdhd734g6papk6wj20y24rm43sk9`                         |
```
Comment (jayantk) [inline pages/documentation/pythnet-price-feeds/cosmwasm.mdx:33]: drive-by: this was confusing

File: components/CosmWasmQuery.tsx [hunk 7]
```diff
@@ -0,0 +1,95 @@
...
+  const { keyValueStore, cosmosChainConfig } = useGlobalContext();
+
+  useEffect(() => {
+    clearResponse();
+  }, [keyValueStore, cosmosChainConfig]);
+
+  const sendTransaction = async () => {
+    const queryJson: JsonObject | undefined = buildQuery(keyValueStore);
+
+    const chain = getCosmosChainFromConfig(cosmosChainConfig.chainId);
```
Comment (cctdaniel) [inline components/CosmWasmQuery.tsx:33]: this function call here takes awhile to load and in the meanwhile nothing is showed in the responsePreface which is not an ideal UX, what do you think about moving `setResponsePreface("Loading...");` up here?

File: components/EvmSend.tsx (hunk 2 above)
Comment (cctdaniel) [reply components/EvmSend.tsx:52]: thanks for the fix

**Review by cctdaniel — APPROVED** [review]
this is great thanks! just one minor comment on the UX for CosmWasmQuery
